graph-db/src/utils/cmr/__test__/fetchPageFromCmr.test.js

```diff
@@ -0,0 +1,107 @@
+const nock = require('nock')
+const AWS = require('aws-sdk')
+let { getEchoToken } = require('../getEchoToken')
+const { fetchPageFromCMR } = require('../fetchPageFromCmr')
+const { expect } = require('@jest/globals')
```

Comment:
You shouldn't need to import this when jest is installed locally as part of your package.json.